website/docs/guides/container_service_extension_4_0_cluster_management.html.markdown

```diff
@@ -0,0 +1,272 @@
+created. In order to craft it, we need to follow these steps:
+
+- First, we need to download a YAML template from the [CAPVCD repository][capvcd_templates].
+  You should choose the template that matches your TKGm OVA. For example, if you uploaded the `ubuntu-2004-kube-v1.22.9+vmware.1-tkg.1-2182cbabee08edf480ee9bc5866d6933.ova`
+  and you want to use it, the template that you need to obtain corresponds to v1.22.9, that is `cluster-template-v1.22.9.yaml`.
+
+- This template requires some extra elements to be added to the `kind: Cluster` block, inside `metadata`. These elements are `labels` and
+  `annotations`, that are required by the CSE Server to be able to provision the cluster correctly. In other words, **cluster creation will fail
+  if these are not added**:
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: ${CLUSTER_NAME}
+  namespace: ${TARGET_NAMESPACE}
+  labels: # This `labels` block should be added, with its nested elements
+    cluster-role.tkg.tanzu.vmware.com/management: ""
+    tanzuKubernetesRelease: ${TKR_VERSION}
+    tkg.tanzu.vmware.com/cluster-name: ${CLUSTER_NAME}
+  annotations: # This `annotations` block should be added, with its nested element
+    TKGVERSION: ${TKGVERSION}
+# ...
+```
+
+- The downloaded template has a single worker pool (to see an example with **two** worker pools, please check the [proposed example][cluster]).
+  If you need to have **more than one worker pool**, you need to add more objects of kind `VCDMachineTemplate`, `KubeadmConfigTemplate` and
+  `MachineDeployment`. In the downloaded template, they look like this:
+
+```yaml
+# ...
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: VCDMachineTemplate
+metadata:
+  name: ${CLUSTER_NAME}-md-0
+# ...
+apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+kind: KubeadmConfigTemplate
+metadata:
+  name: ${CLUSTER_NAME}-md-0
+  namespace: ${TARGET_NAMESPACE}
+# ...
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: MachineDeployment
+metadata:
+  name: ${CLUSTER_NAME}-md-0
+  namespace: ${TARGET_NAMESPACE}
+```
+
+  Notice that the default worker pool is named `${CLUSTER_NAME}-md-0`. To add an extra one, you need to duplicate these
+  blocks and name them differently, for example `name: ${CLUSTER_NAME}-md-1`.
+  If you want to specify a different number of worker nodes per worker pool, you need to modify the original template, otherwise
+  they will share the `${WORKER_MACHINE_COUNT}` placeholder located in the `MachineDeployment` object.
+  The same happens with the **VM Sizing Policy, VM Placement Policy and Storage Profile**.
+  See the explanation below for every placeholder to better understand how to adjust them.
+
+- Now that the YAML template is ready, one needs to understand the meaning of all the placeholders.
+  Below is the explanation of each one of them, you can also check [the working example][cluster] to observe the final result.
```

Comment:
Would it be useful to point out which arguments are optional?

Reply:
Added a note in the optional ones, thanks!